ajax/libs/rlite-router/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "rlite-router",
+  "version": "1.0.82",
+  "description": "A very tiny, fast client-side router",
+  "main": "rlite.js",
+  "directories": {},
+  "scripts": {
+    "test": "jasmine-node test/*.spec.js",
+    "min": "uglifyjs rlite.js --source-map rlite.min.js.map -m -c -o rlite.min.js && echo 'Minified size' && uglifyjs rlite.js | gzip -9f | wc -c"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/chrisdavies/rlite"
+  },
+  "keywords": [
+    "routing"
+  ],
+  "author": "Chris Davies <ferociousturtle@gmail.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/chrisdavies/rlite/issues"
+  },
+  "homepage": "https://github.com/chrisdavies/rlite",
+  "devDependencies": {
+    "jasmine-node": "^1.14.5"
+  },
```

Comment:
Would you please remove `"devDependencies"` and `"homepage"` here? As we don't use `"devDependencies"` and the GitHub is the default `"homepage"` , thanks.